pro_wes/ga4gh/wes/workflow_runs.py

```diff
@@ -406,7 +424,7 @@ def _create_run_environment(
             # populate document
             document.run_log.run_id = run_id
             document.task_id = uuid()
-            document.work_dir = str(work_dir)
+            document.work_dir = work_dir.as_posix()
```

Comment:
I'm curious: Did this trip up mypy or was suggested by it?

If anybode wanted to run proWES on Windows, this would cause problems.

An example from the [documentation](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.as_posix)):

```py
p = PureWindowsPath('c:\\windows')
str(p)
# 'c:\\windows'
p.as_posix()
# 'c:/windows'
```

`c:\\windows` and `c:/windows` are _not_ equivalent, obviously.

Of course, trying to run proWES on Windows is probably not a good idea, and I have no clue how Docker handles Windows file paths on Windows (if at all?). But I'm just wondering why `str()` wouldn't be just fine here (and it wouldn't assume anything about the way the file path notation of the underlying OS).